innovation.css.map

```diff
@@ -0,0 +1 @@
;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AASD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAMD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AASF;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC;;AAJD;EACC;;AAED;EACC","file":"innovationlordmcfuzz.css"}
```

Comment:
Is this file supposed to be added to gitignore? Also, I see "innovationlordmcfuzz" in there.

Reply:
I have to do some research into it. I think it is required but that is without actually knowing why. If it is I'll have to do some sort of solution in the sharecode, which would improve it for them.

Reply:
It looks like it is useful for dev purposes, so I think it should just be ignored and it will be generated locally when you actually need it:
https://css-tricks.com/should-i-use-source-maps-in-production/

I will add to gitignore